lib/pbkit/pbkit.c

```diff
@@ -2234,7 +2234,7 @@ void pb_stop_log(void)
 #endif
 
 
-void pb_end(DWORD *pEnd)
+DWORD *pb_end(DWORD *pEnd)
```

Comment:
Why does this return a value (always `null`) now?

Reply:
So a user can indent all pbkit commands with `p = `; it also makes clear that the command stream pointer is dead after `pb_end()`; this prevents re-use of invalid pointers.

There's no real downside for having this feature as people can simply ignore the return value if they don't want it.

The only downside would be if we decide to use the return value for something else.

Reply:
I don't really see a lot of value there, I think just having the `pb_end()` not indented is more obvious. But it's fine #bikeshed

Reply:
This change has been reverted. Thanks for your input, both of you.